.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
+    name: Build and Test
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
I dont know whether this is actually needed, but is there any reason we would ever want to run against more than one node-version? the matrix option makes it easy to specify an array of versions, if this kind of thing mattered. Again, not sure we actually care about that for our purposes.

Reply:
I have changed this in the latest commit so that we can add different versions to the node-version matrix incase we want to test multiple.